index.js

```diff
@@ -1 +1,33 @@
-console.log("Code your solution!")
+console.log("Code your solution!");
+const form = document.querySelector("form");
+const h1 = document.querySelector("h1");
+const ul = document.querySelector("ul");
+form.addEventListener("submit", (e) => {
+    e.preventDefault(); //keeps page from submitted
+    console.log(e);
+    const li = document.createElement("li");
+    li.textContent = e.target.todo.value; //.todo refres to the name of the input
+    console.log(li); //
+    //!append the created `li` to the list.
+    ul.append(li);
+    form.reset()
+    // //! create error
+    // const errMessage = document.getelementById(errorMsg)
+    //! When the user clicks on one of the `li` items, the item should be crossed out
+    li.addEventListener("click", (event) => {
```

Comment:
I think moving this event listener outside of the form event listener might get rid of the error you are getting.